samples/boards/thingy52_nrf52832/pdm/src/main.c

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2020 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <device.h>
+#include <drivers/audio/pdm.h>
+#include <drivers/gpio.h>
+
+#define THINGY52_MIC_PWR_PIN 9
+#define MIC_PWR_ON 1
```

Comment:
To make this compatible with other nRF boards, its best to make these configurable and optional.